libnmstate/nm/nmclient.py

```diff
@@ -22,19 +22,17 @@
 
 import six
 
+import gi                            # pylint: disable=import-error
```

Comment:
I see it was there before but IMHO it is bad practice to align comments, since adding a new, longer line might require to adjust all the comments in unrelated lines.